### DISCUSSION TITLE
Expression builder?

### DISCUSSION THREAD

**Discussion by kpdemetriou** [geldata/gel#2812] [Q&A] [answered]
Are there any plans to support something like `SQLAlchemy`'s expression API in the Python driver?

**Comment by 1st1** [comment, marked as answer]
We're working on a proper query builder for TypeScript and will soon have a technical preview of it. We'll focus our efforts to create a similar library for Python+mypy right after that.

**Reply by kpdemetriou** [reply]
Great to hear this is in the pipeline. I'm looking forward to trying out EdgeDB in an upcoming microservice and this is the last blocker. Perhaps access control as well, but there are ways around that (as ES has proved).
